object_detector:
  confidence_min: 0.3
  target_class: chair
  input_img_size: 640